forge-gui/res/cardsfolder/upcoming/bladewhip_transmogrant.txt

```diff
@@ -0,0 +1,11 @@
+Name:Bladewhip Transmogrant
+ManaCost:2
+Types:Artifact Creature Zombie
+PT:3/1
+K:CARDNAME can't block.
+A:AB$ ChangeZone | Cost$ 4 B B | Origin$ Graveyard | Destination$ Battlefield | ReduceCost$ X | WithCountersType$ P1P1 | ActivationZone$ Graveyard | SpellDescription$ Return CARDNAME from the graveyard to the battlefield with a +1/+1 counter on it. This ability costs 4 less to activate if an opponent controls four or more nonbasic lands.
+SVar:X:Count$Compare Y GE4.4.0
+SVar:Y:Count$Valid Land.OppCtrl+nonBasic
```

Comment:
I don't think this works... I think ONE opponent (of however many you may have) must control 4
As this is, if one opponent controlled 2 nonbasics and another controlled 3, you'd get the cost reduction.

Reply:
Hmm I copied this from a similar card that also said "an opponent". Will test with multiple opponents later

Reply:
If it's an older card, it may just have never been refactored for multiplayer - we find stuff like that from time to time